lisa/base_tools/cat.py

```diff
@@ -21,9 +21,13 @@ def read(
     ) -> str:
         # Run `cat <file>`
         result = self.run(
-            file, force_run=force_run, sudo=sudo, shell=True, no_debug_log=no_debug_log
+            file,
+            force_run=force_run,
+            sudo=sudo,
+            shell=True,
+            no_debug_log=no_debug_log,
+            expected_exit_code=0,
         )
-        result.assert_exit_code(message=f"Error : {result.stdout}")
```

Comment:
As discussed this stdout/stderr needs to be handled separately. Other changes look good to me

Reply:
reverted